src/main/java/uk/gov/justice/digital/zone/Zone.java

```diff
@@ -3,34 +3,21 @@
 import io.delta.tables.DeltaTable;
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.Row;
-import org.apache.spark.sql.SaveMode;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import uk.gov.justice.digital.service.model.SourceReference;
+import uk.gov.justice.digital.service.DataStorageService;
 
 import java.util.List;
 
 import static org.apache.spark.sql.functions.col;
 import static uk.gov.justice.digital.job.model.Columns.*;
 
-public abstract class Zone {
+public abstract class Zone extends DataStorageService {
```

Comment:
This seems a bit strange in terms of inheritance. Better to have this wired in via DI as discussed.

Reply:
removed